src/widgets/DataTable/Row.js

```diff
@@ -47,12 +47,12 @@ const Row = React.memo(
     }
 
     const { adjustToTop } = useContext(RefContext);
-
+    const onPressRow = useCallback(() => onPress(rowData));
```

Comment:
does this need an array? With `rowData` in it? ( I'm not sure, hence the question.. )

Reply:
ya I guess I missed a lot ..

Reply:
yeah, as many as 4 small things in 100s of lines of code changes. 🤔